nilearn/decoding/tests/test_space_net.py

```diff
@@ -187,6 +187,20 @@ def test_tv_regression_3D_image_doesnt_crash():
                      penalty="tv-l1", is_classif=False, max_iter=10).fit(X, y)
 
 
+def test_graph_net_classifier_score(C=.01, tol=1e-10, zero_thr=1e-4):
```

Comment:
I'd simply the test. Ideally the prototype should be `test_*()`, with no args or kwargs. Also, must of the parameters you setting in SpaceNetClassifier are idea irrelevant, or are already the default.